src/fragments/lib/auth/common/mfa/flows.mdx

```diff
@@ -0,0 +1,214 @@
+
+import iosConfirmSignIn from "/src/fragments/lib/auth/ios/totp/confirm_sign_in.mdx";
+
+<Fragments fragments={{ios: iosConfirmSignIn}} />
+
+### Enabling TOTP after a user is signed in
+
+TOTP MFA can be set up after a user has signed in. This can be done when the following conditions are met:
+
+- MFA is marked as **Optional** or **Required** in the Cognito User Pool
+- TOTP is marked as an enabled MFA method in Cognito user pool
+
+TOTP can be set up by calling the `setUpTOTP` and `verifyTOTPSetup` APIs in the `Auth` category.
+
+Invoke the `setUpTOTP` API to generate a `TOTPSetupDetails` object which should be used to configure an Authenticator app like Microsoft Authenticator or Google Authenticator. `TOTPSetupDetails` provides a helper method called `getSetupURI` which generates a URI that can be used, for example, in a button to open the user's installed Authenticator app. For more advanced use cases, `TOTPSetupDetails` also contains a `sharedSecret` which can be used to either generate a QR code or be manually entered into an Authenticator app.
+
+that contains the `sharedSecret` which will be used to either to generate a QR code or can be manually entered into an Authenticator app.
+
+**`setUpTOTP` API**
+
+import iosSetUpTOTP from "/src/fragments/lib/auth/ios/totp/set_up_totp.mdx";
+
+<Fragments fragments={{ios: iosSetUpTOTP}} />
+
+Once the Authenticator app is set up, the user must generate a TOTP code and provide it to the library. Pass the code to `verifyTOTPSetup` to complete the TOTP setup process.
+
+**`verifyTOTPSetup` API**
+
+import iosVerifyTOTPSetup from "/src/fragments/lib/auth/ios/totp/verify_totp_setup.mdx";
+
+<Fragments fragments={{ios: iosVerifyTOTPSetup}} />
+
+### Recovering from a lost TOTP device
+
+<Callout warning>
+
+If a user loses access to their TOTP device, they would need to contact an administrator to help get access to their account. Based on the Cognito User Pool configuration, the administrator can use the [AdminSetUserMFAPreference](https://docs.aws.amazon.com/cognito-user-identity-pools/latest/APIReference/API_AdminSetUserMFAPreference.html) to either change the MFA preference to a different MFA method or to disable MFA for the user.
+
+</Callout>
+
+In a scenario where MFA is marked as **Required** in Cognito User Pool and another MFA method is not set up, the administrator would need to first initiate an [AdminUpdateUserAttributes](https://docs.aws.amazon.com/cognito-user-identity-pools/latest/APIReference/API_AdminUpdateUserAttributes.html) call and update the user’s phone number attribute. Once this is complete, the administrator can continue changing the MFA preference to SMS as suggested above.
+
+## Setting a user's preferred MFA option
+
+<Callout>
+
+The APIs for fetching and updating MFA preferences are only available through AWS Cognito Auth plugin. The examples usage of the API's using the plugin is provided below. 
```

Comment:
why is this important to call out here? Let us remove it. If this is because of the Auth0 plugin, I don't think this callout should be here.

Reply:
It's called out because they're not available on the category, only the plugin.

Reply:
My question here is about why do we need to have that be a callout? Can we just not provide the code snippet and describe the behavior and move on? Why would a developer care?